### ISSUE TITLE
TF 1.14: meta optimizer Invalid argument:

### ISSUE THREAD

**Issue by UrosTo** [tensorflow/tensorflow#31774]
Hello guys,

For my school paper I am doing fine tuning of some neural networks and one neural network I am using is tiny-yolo from [here](https://github.com/qqwweee/keras-yolo3), when I start the training script it loads the GPU but when it starts this is what I get 

![](https://user-images.githubusercontent.com/54294015/63305631-f549d000-c2e7-11e9-8b32-b45368309ebf.PNG)

The code is using Adam optimizer. From what I see while tracking the GPU/CPU/RAM usage of the application Cuda cores are used every few seconds and only 10% of them. I am not sure if this is because of the errors or what (since on different networks the ones I used for learning it uses 60% but only since I capped it at that)

Is this a known bug and how can I fix it? Or is the slowness of the training not related to this?

**Comment by gadagashwini** [comment]
Please provide details about what platform you are using (operating system, architecture). Also include your TensorFlow version. Also, did you compile from source or install a binary?

Make sure you also include the exact command if possible to produce the output included in your test case. If you are unclear what to include see the issue template displayed in [the Github new issue template](https://github.com/tensorflow/tensorflow/issues/new/choose).

We ask for this in the issue submission template, because it is really difficult to help without that information. Thanks!

**Comment by UrosTo** [comment]
Code: I am using code written by someone else, the link to it can be found in the original post. The scrip I am running is train.py (with custom dataset for training, so the location of train.txt is different but the format is the same). He is using Adam optimizer

OS: Windows 10 with Anaconda for python
TF: TensorFlow-GPU installed with -pip command, version of tensorflow is 1.14.0
Python version is 3.7.3
GPU: GTX 1060 with 6GB RAM
Cuda compilation tools, release 10.0, V10.0.130
CPU: AMD FX-6300-Six-Core Processor at 3.5 GHz

Not sure if more info is needed.

**Comment by gadagashwini** [comment]
@UrosTo Thanks for details.
Please provide the minimal code snippet to reproduce the reported issue or provide the link to the original post. Thanks!

**Comment by UrosTo** [comment]
Not sure if this is what you are asking for, I am running this [train.py](https://github.com/qqwweee/keras-yolo3/blob/master/train.py)
I should add that I am training tiny-YOLO.

**Comment by gadagashwini** [comment]
@UrosTo, I tried replicating the issue, looks like the repository has some dependencies. Will it be possible to provide the minimal code snippet to reproduce the reported issue. Thanks!

**Comment by UrosTo** [comment]
@gadagashwini I am not really sure what you are asking me to provide now.
Can you please explain a bit?

**Comment by gadagashwini** [comment]
@UrosTo, We need a complete code to investigate the issue. It would be good if you provide the complete code with all the operation. Thanks!

**Comment by UrosTo** [comment]
@gadagashwini oh sorry I got confused since all of the neural network code was linked in the original post. Here is the github link to the [code](https://github.com/qqwweee/keras-yolo3).
The only thing I am running is [train.py](https://github.com/qqwweee/keras-yolo3/blob/master/train.py) that I will link again.
Hope this helps in pinpointing the problem. If you need more please let me know

**Comment by jvishnuvardhan** [comment]
@UrosTo Sorry for the late response. Is this still an issue? If yes, could you please post it in the `yolo3` repository [here](https://github.com/qqwweee/keras-yolo3/issues). Thanks

Please close this issue, once you post the issue in `yolo3` repo. Thanks!

**Comment by UrosTo** [comment]
Unfortunately cause of some problems with the PC I was doing this on I can not confirm if the problem is still there or not. I lost the code I used and all of the data I used, and cause of reinstalling a lot of stuff I don't know if now I got everything the same as it was.
Since that is the case, and I can no longer recreate the same environment to check if the problem is still there, I will close this issue.
